webapp/packages/plugin-administration/src/ConfigurationWizard/ServerConfiguration/IServerConfigurationFormPartState.ts

```diff
@@ -11,6 +11,7 @@ const ServerConfigurationFormPartStateConfigSchema = schema.object({
   adminCredentialsSaveEnabled: schema.boolean().optional(),
   adminName: schema.string().optional(),
   adminPassword: schema.string().optional(),
+  adminPasswordRepeat: schema.string().optional(),
```

Comment:
i think we can validate it only on the component level and not add this form state / or form part checks

Reply:
We clear password fields after refresh. I believe it's the right logic. Why should it work after refresh?

Reply:
If we clear it, then okay. The password can be remembered (between refreshes), and validation will not pass if the repeated password is not filled in.

Reply:
It seems there is an issue with the video. The password field had validation in place and the validation was triggered every time the user clicked the 'Next' button. If it is regression, we should fix it first

Reply:
https://github.com/user-attachments/assets/fb2e9c4d-5756-4973-aa9a-857447eafa5f

Reply:
Maybe we can subscribe to the part submit or validate event from AdminForm component and interrupt execution if passwords dont match